README.md

```diff
@@ -1,207 +1,16 @@
-
-Many of our models will have attributes that are required for our application to use and display data consistently. Each model will have attributes with requirements for a valid record. The requirements are summarized below:
-
-#### User
-
-- Username must be present
-- Username must be unique
-- Email Address must be present
-- Email Address must be unique
-- Password Confirmation must match Password
-
-#### Product
-
-- Name must be present
-- Name must be unique
-- Price must be present
-- Price must be a number
-- Price must be greater than 0
-- Product must belong to a User
-
-#### Order
-
-- An Order must have one or more Order Items
-
-#### OrderItem
-
-- Must belong to a Product
-- Must belong to an Order
-- Quantity must be present
-- Quantity must be an integer
-- Quantity must be greater than 0
-
-#### Review
-
-- Rating must be present
-- Rating must be an integer
-- Rating must be between 1 and 5
-
-## Submission Guidelines
-Your final project must be deployed to [Heroku](http://heroku.com). Your team will open a single pull request for the entire project. Include the link to your Heroku deployment in the PR's description.
-
-Your `README` file on your Github repo should contain a link to your live site.
-
-## Team Leaders
-Each team will have team leaders who are responsible for keeping track of each team member's contributions. Rotate leader roles at the beginning of each week; every team member should be in at least one leader role during the project.
-
-- Stand Up Leader
-  - Notifies team members about meeting schedule and ensures that everyone is present and ready
-  - Takes notes about each person's daily report in Stand Up
-  - Keeps the meeting moving
-- Task Leader
-  - Leads discussion on task assignment
-    - Decide if a task should be completed alone or in a pair
-    - Assign tasks based on...
-      - Individual comfort
-      - Desire
-      - Ability
-  - Ensures the task list stays up to date
-
-## Stand Up Meetings
-The Project Manager for your team will determine the timing for all Stand Up meetings. Because PMs are managing multiple projects at once this time will be different for each team, and may change from day to day. The meeting schedule will be communicated to the Stand Up Leader as soon as it is determined.
-
-At the end of each day, your team's assigned Project Manager will review the Trello board for all tasks discussed during that day's Stand Up meeting.
-
-## Weekly Demos
-In a real world work environment, a team's success is measured by their product as opposed to each individual's contribution.
-
-Each team will present their progress and respond to questions from their Project Manager each Friday. Every team member will participate in these demos; the PM will ask specific questions regarding 1) the team's progress and plan for completing the project, 2) the technical decisions and implementation, and 3) every team member's understanding of the underlying technical structures.
+# Wetsy

```

Comment:
I like this Wetsy version of the README! The only thing I would suggest to add at this point is maybe a section with setup instructions, so that people whom would like to use your project will need to know what steps they should take to get going, including cloning and forking.